tools/runners/run-test-suite-test262.py

```diff
@@ -91,14 +85,14 @@ def update_exclude_list(args):
     # Tests pass in strict-mode but fail in non-strict-mode (or vice versa) should be considered as failures
     passing_tests = passing_tests - failing_tests
 
-    with open(args.excludelist_path, 'r+', encoding='utf8') as exclude_file:
+    with open(args.excludelist_path, 'rb+') as exclude_file:
```

Comment:
I moved the encoding paramter to 
latter

https://github.com/jerryscript-project/jerryscript/pull/4856/files/d59a19a0ddb6a4f8bdf8571c3fea2fedfcd2a636#diff-332d120efcdcc358eca735131630eb1b8d9f49f8d0932ee23ea3477a698cd48aL101

for avoid decoding failure

Reply:
Oh, I gotcha it, it's because when I create this MR is for support python2, if python 2 support is dropped, then I can use errors='ignore' as open parameter

https://docs.python.org/2/library/functions.html#open

https://docs.python.org/3/library/functions.html#open

Reply:
Yes, python2 is not supported anymore.